fl4health/model_bases/apfl_base.py

```diff
@@ -4,8 +4,10 @@
 import torch
 import torch.nn as nn
 
+from fl4health.model_bases.partial_layer_exchange_model import PartialLayerExchangeModel
 
-class ApflModule(nn.Module):
+
+class ApflModule(PartialLayerExchangeModel):
```

Comment:
Having models that exchange a subset of their weights inherit from an abstract based class that forces the implementation of the `layers_to_exchange` function. This is just a formalism for now, but should be useful in future iterations of the parameter exchanger mechanisms.e